File: front/src/entity/record/source.ts
export class Source {
  library: string;
  url: string;

  getLibrary(): string {
    return this.library;
  }

  setLibrary(value: string) {
    this.library = value;
  }

  getUrl(): string {
    return this.url;
  }

  setUrl(value: string) {
    this.url = value;
  }
}

File: back/src/main/java/org/lendi/umtapo/entity/record/simple/Source.java
package org.lendi.umtapo.entity.record.simple;

import com.fasterxml.jackson.annotation.JsonInclude;

/**
 * Source entity.
 */
@JsonInclude(JsonInclude.Include.NON_NULL)
public class Source {
    private String library;
    private String url;

    public String getLibrary() {
        return library;
    }

    public void setLibrary(String library) {
        this.library = library;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}
